tools/crm_rule.c

```diff
@@ -396,7 +392,14 @@ main(int argc, char **argv)
      */
     switch(options.mode) {
         case crm_rule_mode_check:
-            rc = crm_rule_check(out, data_set, options.rule, rule_date);
+            for (char **s = options.rules; *s != NULL; s++) {
+                rc = crm_rule_check(out, data_set, *s, rule_date);
+
+                if (rc > pcmk_rc_ok) {
```

Comment:
Why treat these errors differently? If a user mistypes one rule name, I'd think they just want to see that error message and continue with any other rules. Even if all the rules will fail with the same error, that seems worthwhile to know.

Exiting with the last status is acceptable but feels a bit off. Maybe exit OK only if all rules were OK (or in range), and otherwise use last non-OK status?

Reply:
Fixed.